src/asm_parser.rs

```diff
@@ -5,11 +5,12 @@
 // file of the crate. Do not expect to find those comments in the documentation of the crate.
 
 //! This module parses eBPF assembly language source code.
-
```

Comment:
Nit: `cargo fmt` doesn't seem to remove this blank line for me, and I'd avoid removing it because the comment is not related to the imports below.